lib/middleman-prismic/commands/prismic.rb

```diff
@@ -29,40 +23,68 @@ def self.exit_on_failure?
       end
 
       def prismic
-        ::Middleman::Application.server.inst
-        reference = MiddlemanPrismic.options.release
-
-        Dir.mkdir('data') unless File.exists?('data')
-
-        FileUtils.rm_rf(Dir.glob('data/prismic_*'))
+        create_directories
+        output_available_documents
+        output_references
+        output_custom_queries
+      end
 
-        api = ::Prismic.api(MiddlemanPrismic.options.api_url)
-        response = api.form('everything').submit(api.ref(reference))
+      private
 
-        available_documents = []
-        response.each {|d| available_documents << d.type}
+      def create_directories
+        if File.exists?("#{DATA_DIR}/prismic")
+          FileUtils.rm_rf(Dir.glob("#{DATA_DIR}/prismic"))
+        end
 
-        available_documents.uniq!
+        Dir.mkdir("#{DATA_DIR}") unless File.exists?("#{DATA_DIR}")
```

Comment:
Do we need the string interpolation? i.e. can we replace `"#{DATA_DIR}"` with `DATA_DIR`?